openwisp_utils/admin_theme/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.contrib.admin.apps import AdminConfig
+from ..settings import CUSTOM_ADMIN_SITE
+
+class OpenwispAdminConfig(AdminConfig):
+    default_site = CUSTOM_ADMIN_SITE
```

Comment:
don't do this, revert the changes in this file and follow my subsequent suggestions